change-beta/@azure-communication-react-b037427e-c8d3-49dd-b4c2-f7fabef4e35b.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
```

Comment:
Do we need chnagelog for Storybook (as it's not part of communication-react package)?

Reply:
No need for changelog. Would be safe to remove

Reply:
thanks